python/pyproject.toml

```diff
@@ -4,34 +4,70 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "aac-puml"
-version = "0.1.3"
+version = "0.1.4"
 authors = [
-	{ name="crazynewidea", email="asdfasdf@mail.com" },
-    { name="lizzcondrey", email="asdfasdfd@mail.com" },
-    { name="bunchw", email="asdfdas@mail.com" }
+	{name="AaC Dev Team", email="asdfasdaf@email.com"}
 ]
 description = "PlantUML diagram output plugin for AaC (Architecture-as-Code) project"
 requires-python = ">= 3.9.13"
-dependencies = [
-	"aac>=0.4.3"
-]
 readme = "README.md"
 license = {text = "MIT License"}
-keywords = ["SysML", "Architecture-as-Code", "System Engineering"]
+keywords = ["SysML", "Architecture-as-Code", "System Engineering", "MBSE"]
 classifiers = [
 	"Programming Language :: Python :: 3",
 	"License :: OSI Approved :: MIT License",
 	"Operating System :: OS Independent",
+    "Topic :: Scientific/Engineering"
+]
+dependencies = [
+	"aac >= 0.4.3",
+]
+[project.optional-dependencies]
+doc = [
+	"sphinx >= 7.2.6",
+	"sphinx-autobuild ~= 2021.3.14",
+	"sphinx_contributors >= 0.2.7",
+	"sphinx-copybutton >= 0.5.2",
+	"sphinxcontrib-applehelp ~= 1.0.2",
+    "sphinxcontrib-devhelp ~= 1.0.2",
+    "sphinxcontrib-htmlhelp ~= 2.0.0",
+    "sphinxcontrib-jsmath ~= 1.0.1",
+    "sphinxcontrib-qthelp ~= 1.0.3",
+    "sphinxcontrib-serializinghtml ~= 1.1.9",
```

Comment:
Something happened with this indent it looks like

Reply:
These indents look normal on gitpod and in the code section of the branch, at least for me.  Not sure why it looks weird here.

Reply:
Checked in code view and agree. So diff is just being weird to display it seems.